.ruby-version

```diff
@@ -1 +1 @@
-2.7
+3.0
```

Comment:
is the plan move this back to 2.7 before merging, but just make sure tests pass on 3.0?

Reply:
whoops, yep, thank you